sentry-samples/sentry-samples-android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
already exists https://github.com/getsentry/sentry-java/tree/main/gradle/wrapper

Reply:
Ahh, i opened the sample subfolder instead of the main one, thanks!